openedx_events/learning/signals.py

```diff
@@ -297,7 +297,7 @@
 # .. event_data: DiscussionThreadData
 # .. event_warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
 FORUM_THREAD_CREATED = OpenEdxPublicSignal(
-    event_type="org.openedx.learning.thread.created.v1",
+    event_type="org.openedx.learning.forum.thread.created.v1",
```

Comment:
Does this need to be changed?
Should this emit a `FORUM_THREAD_CREATED_V2` signal?

I imagine this will break existing code

Reply:
Oh yes, as I mentioned in the cover letter those changes are there for testing. That should be addressed in a follow up PR.